aws/cloudformation/components/database.yml.erb

```diff
@@ -11,6 +11,19 @@
       Parameters: <%=values.compact.to_json%>
 <%end -%>
 
+<%  db_secrets = [nil, 'application-writer', 'readonly'].map do |user| -%>
+  DatabaseSecret<%=secret = user&.underscore&.camelcase%>:
```

Comment:
I'm a little confused by this code. Looks like you're making three resources, but I don't follow how the string manipulation is working. I assume you're trying to get to a camel cased string with no hyphens or underscores.

Reply:
I guess this isn't new code, so i assume it works 😆

Reply:
Yup! I think the camel-casing and other string formatting is to take existing SQL login names and convert them into fairly standard CloudFormation template Resource names.